index.html

```diff
@@ -176,50 +241,80 @@ <h3 id="matters-considered-by-the-placement-rules">Matters considered by the pla
-      However, since ruby is not restricted to 1/2,
-      this document uses the size the base character as the reference.
-    </aside>
-    <aside class=note title="Size of ruby characters">
-      In movable type typesetting,
-      when ruby characters of size 3.5 points were not available,
-      based characters of 7 points where sometimes paired
-      with ruby characters of 4 points.
-      Also, ruby associated with large base characters,
-      such as those in titles,
-      are sometimes smaller than 1/2.
-    </aside>
-  </li>
-  <li>While there are cases of ruby on both sides of the base string exist,
-    the method defined here only handles ruby on one side.
-    Handling both sides is left as a future exercise.</li>
+    and will use the same logic in either case.
+    <spanclass="checkme">
+    Specifically, the center of the ruby string and of the base character 
+    string are aligned in the inline direction for mono-ruby. 
+    </span>
+    </li>
+  <li><span class="checkme">Two step processing method is taken. 
+    In the first step, processing of layout only considers about the ruby 
+    string and the base character string (collectively call both of them as 
+    the ruby block in this note), to decide relative position of the ruby 
+    string and the ruby base character string. 
+    In the second step, processing of layout decides a position of the ruby 
+    base character string in a line, with consideration of preceding and 
+    following characters.
+    On the other word, the relative position of the ruby string and the 
+    ruby base character string decided in the first step is not modified 
+    regarding of any preceding and following characters. 
+    Also, this note does not take a method to align the first or last 
+    character of the ruby base character string to the line head or the line 
+    end, by modifying the relative position of the ruby string and the ruby 
+    base character string when the ruby base character string is placed at 
+    the line head or the line end. 
+    Summarizing the above, resulting positionings by the first step are not 
+    modified by the second step at all. 
+    </span>
+    </li>
+  <li>
+    <span class="checkme">
+    Although there are some cases with multiple methods of layout in [[JLREQ]] 
+    and JIS X 4051 [[JISX4051]], this note describes only one method based on 
+    policies described above. 
+    Also methods described in this note are mostly chosen from ones provided 
+    in JIS X 4051 [[JISX4051]]. 
+    In some cases, this note picks optional methods to be allowed as 
+    implementation defined, such that protruding ruby string is not laid over 
+    any preceding and following Kana characters. 
+    </span>
+    </li>
+  <li>
+    <span class="checkme">
+    There is a demand to use larger (or smaller) font size for ruby string. 
+    In this note, a default font size of ruby string is set to half of a 
```

Comment:
a font size -> the font size

(Same below.)

Reply:
fixed (two items)

Reply:
This doesn't seem to be fixed. Could you check again?

Reply:
(could you reload?? - at least github says this part is 'Outdated')

Reply:
According to GitHub, this part is "Outdated" indeed, but in your branch (https://github.com/himorin/simple-ruby/blob/202004-jlreqtf-update/index.html) I see it's still not changed.

Reply:
ah! I did change for the later one, but missed the first one.. sorry.